charts/renku/templates/configmap.yaml

```diff
@@ -63,3 +64,84 @@ data:
     # configure the logout redirect
     curl -f -is -X PUT -H "Private-token: ${GITLAB_SUDO_TOKEN}" \
       ${GITLAB_SERVICE_URL}/api/v4/application/settings?after_sign_out_path={{ template "http" . }}://{{ .Values.global.renku.domain }}/auth/realms/Renku/protocol/openid-connect/logout?redirect_uri=${GITLAB_URL}
+
+  # Init scripts that populate /docker-entrypoint-initdb.d
+
+  # Unashamedly copied from: https://github.com/docker-library/postgres/blob/master/9.6/docker-entrypoint.sh
+  init-postgres.sh: |-
```

Comment:
This is going to keep crashing and recreating new pods to run the job until the server is available -- a better solution might be to wait for the server to be up before continuing. See the other post-install script: https://github.com/SwissDataScienceCenter/renku/pull/293/files#diff-0f15bad1c1882fee30ebc792c5a38350R24

Reply:
That job is doing a curl to wait. Would need an equivalent psql command to ping.

Reply:
yes I wasn't suggesting that what the other script does is ideal ;)  
But IMO the script should do the waiting, not rely on k8s to just keep relaunching the pod.

Reply:
You could also install curl in the postgres container?

Reply:
@rokroskar , @leafty : eh, what's the action point here? r we settled on a curl trick?

Reply:
@fgeorgatos it was fixed in 864f2a0